app/models/miq_shortcut.rb

```diff
@@ -3,11 +3,17 @@ class MiqShortcut < ApplicationRecord
   has_many :miq_widgets, :through => :miq_widget_shortcuts
 
   def self.seed
-    names = []
+    db_records = all.to_a
+    db_data = db_records.index_by(&:name)
+
+    if db_records.size != db_data.size
+      names = db_records.group_by(&:name).select { |n, v| v.size > 1 }.map(&:first)
+      _log.warn("Duplicate seeds for names: #{names.join(",")}")
+    end
```

Comment:
So this warning is only going to go off when re-running the `MiqShortcut.seed` after the data has been entered, correct?  So this might not catch things right away. Should we maybe be running this logic on the `seed_data` keys instead?